### PR TITLE
EI-887 Move tf module to github

### PR DESCRIPTION
**Pull request by saalimk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EI-887

### Change description ###

EI-887 Move tf module to github

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: CPP Azure route table module
+trigger: none
+pr:
+- dev/EI-887
```
Comment (gunnertwin) [inline azure-pipelines.yml]: Should this be `main`?

File: interpolated-defaults.tf [hunk 2]
```diff
@@ -1,2 +1,2 @@
 data "azurerm_subscription" "current" {
```
Comment (gunnertwin) [inline interpolated-defaults.tf]: Is this needed? I couldn't find it in use anywhere

**Review by gunnertwin — COMMENTED** [review]
A couple of comments added

File: azure-pipelines.yml (hunk 1 above)
Comment (saalimk) [reply azure-pipelines.yml]: yes I will update that

File: interpolated-defaults.tf (hunk 2 above)
Comment (saalimk) [reply interpolated-defaults.tf]: deleted as not required.

**Review by gunnertwin — APPROVED** [review]

**Pull request merged**